### ISSUE TITLE
[Error] String too big

### ISSUE THREAD

**Issue by FPSUsername** [MrUniverse44/PixelMOTD#3]
I'm running on 1.16.5 [Paper](https://papermc.io/) (build [629](https://papermc.io/api/v2/projects/paper/versions/1.16.5/builds/629/downloads/paper-1.16.5-629.jar)) combined with plugins including [TCP Shield](https://github.com/TCPShield/RealIP).

When I'm in the server browser, the server seems offline (not reachable), however, I can still join it and play fine.
The server spits out this huge warning:

```
[13:29:55] [Netty Server IO #1/ERROR]: io.netty.handler.codec.EncoderException: String too big (was 76937 bytes encoded, max 32767)
[13:29:55] [Netty Server IO #1/WARN]: io.netty.handler.codec.EncoderException: String too big (was 76937 bytes encoded, max 32767)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketDataSerializer.a(PacketDataSerializer.java:393)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketDataSerializer.a(PacketDataSerializer.java:385)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketStatusOutServerInfo.b(PacketStatusOutServerInfo.java:34)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketEncoder.encode(PacketEncoder.java:46)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketEncoder.encode(PacketEncoder.java:15)
[13:29:55] [Netty Server IO #1/WARN]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[13:29:55] [Netty Server IO #1/WARN]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[13:29:55] [Netty Server IO #1/WARN]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[13:29:55] [Netty Server IO #1/WARN]: 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelInjector.encode(ChannelInjector.java:529)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelInjector.access$100(ChannelInjector.java:71)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelInjector$1.encode(ChannelInjector.java:269)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelInjector$1.write(ChannelInjector.java:275)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:790)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:758)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.writeAndFlush(AbstractChannelHandlerContext.java:808)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.DefaultChannelPipeline.writeAndFlush(DefaultChannelPipeline.java:1025)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannel.writeAndFlush(AbstractChannel.java:294)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelProxy.writeAndFlush(ChannelProxy.java:327)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.b(NetworkManager.java:291)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.dispatchPacket(NetworkManager.java:267)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.sendPacket(NetworkManager.java:243)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.sendPacket(NetworkManager.java:229)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.destroystokyo.paper.network.StandardPaperServerListPingEventImpl.processRequest(StandardPaperServerListPingEventImpl.java:107)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketStatusListener.a(PacketStatusListener.java:149)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketStatusInStart.a(SourceFile:22)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.PacketStatusInStart.a(SourceFile:8)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.a(NetworkManager.java:180)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.channelRead0(NetworkManager.java:169)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.NetworkManager.channelRead0(NetworkManager.java:48)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:302)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at com.comphenix.protocol.injector.netty.ChannelInjector$2.channelRead(ChannelInjector.java:290)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at net.minecraft.server.v1_16_R3.LegacyPingHandler.channelRead(LegacyPingHandler.java:121)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.handler.flush.FlushConsolidationHandler.channelRead(FlushConsolidationHandler.java:152)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
[13:29:55] [Netty Server IO #1/WARN]: 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[13:29:55] [Netty Server IO #1/WARN]: 	at java.base/java.lang.Thread.run(Unknown Source)
```

Tested (and warns) with:
[8.8.6-UnfinishedAlpha3-N20](https://github.com/MrUniverse44/PixelMOTD/releases/tag/8.8.6-UnfinishedAlpha3-N20)
[8.8.6-UnfinishedAlpha3-N18](https://github.com/MrUniverse44/PixelMOTD/releases/tag/8.8.6-UnfinishedAlpha3-N18)

Works fine on [8.8.6-PreAlpha1](https://www.spigotmc.org/resources/bungee-spigot-%E2%9E%A5-pixel-motd.37177/download?version=367660)

**Comment by MrUniverse44** [comment]
This issue will be fixed in this github: https://github.com/sebastnchan/PixelMOTD/

**Comment by MrUniverse44** [comment]
This issue was fixed in 9.0.0.
New PixelMOTD Github Repo: 
https://github.com/MrUniverse44/PixelMOTD2/
